connectors/sources/generic_database.py

```diff
@@ -447,17 +447,18 @@ async def fetch_rows(self, schema=None):
         """
         tables_to_fetch = await self.get_tables_to_fetch(schema)
         if self.database in self.tables_to_skip.keys():
-            tables_to_fetch = [
-                *(set(tables_to_fetch) - set(self.tables_to_skip[self.database]))
-            ]
+            tables_to_fetch = set(tables_to_fetch) - set(
+                self.tables_to_skip[self.database]
+            )
+        tables_to_fetch = list(tables_to_fetch)
```

Comment:
Why change this?

Reply:
As type of `tables_to_fetch` is map, we need to convert it to list. also to remove multiple type conversion we remove it from the L450